src/index.ts

```diff
@@ -110,7 +165,7 @@ const processFile = (filePath: string, isIndex: boolean): string => {
                 <link rel="stylesheet" href="${
                   path.relative(path.dirname(filePath), input) || './'
                 }/index.css"> 
-                <title>${title || path.basename(filePath, '.txt')}</title>`;
+                <title>${title || path.basename(filePath, isMd ? '.md' : '.txt')}</title>`;
```

Comment:
You can write `path.basename(filePath, path.extname)` to remove the extension from name.

Reply:
changed